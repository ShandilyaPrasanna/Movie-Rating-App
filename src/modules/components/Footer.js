import React from "react";
import { makeStyles } from "@material-ui/core/styles";
import Container from "@material-ui/core/Container";

// Footer Component
const useStyles = makeStyles(theme => ({
  root: {
    display: "flex",
    flexDirection: "column",
    minHeight: "73vh"
  },
  footer: {
    padding: theme.spacing(2),
    marginTop: "auto",
    fontSize: "18px",
    backgroundColor: "transparent"
  }
}));

export default function Footer() {
  const classes = useStyles();
  return (
    <div className={classes.root}>
      <footer className={classes.footer}>
        <Container maxWidth="sm">
          Follow us on social media Sites :- <a>Facebook</a> <a>Twitter</a>{" "}
          <a>Instagram</a> <a>Linkedln</a>
        </Container>
      </footer>
    </div>
  );
}
