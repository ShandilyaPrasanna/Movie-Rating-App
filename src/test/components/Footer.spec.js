import React from "react";
import ReactDOM from "react-dom";
import toJson from "enzyme-to-json";
import { shallow, configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import Footer from ".../../modules/components/Footer";

jest.mock("react-dom");

configure({ adapter: new Adapter() });
describe("Footer Test", () => {
  let tree;
  beforeEach(() => {
    tree = shallow(<Footer />);
  });

  it("renders Footer ", () => {
    expect(toJson(tree)).toMatchSnapshot();
  });
});
