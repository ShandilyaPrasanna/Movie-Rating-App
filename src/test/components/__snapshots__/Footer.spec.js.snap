// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Test renders Footer  1`] = `
<div
  className="makeStyles-root-1"
>
  <footer
    className="makeStyles-footer-2"
  >
    <WithStyles(ForwardRef(Container))
      maxWidth="sm"
    >
      Follow us on social media Sites :- 
      <a>
        Facebook
      </a>
       
      <a>
        Twitter
      </a>
       
      <a>
        Instagram
      </a>
       
      <a>
        Linkedln
      </a>
    </WithStyles(ForwardRef(Container))>
  </footer>
</div>
`;
